src/main/java/Duke.java

```diff
@@ -1,10 +1,238 @@
+        String sign_off = "\t" + SECTION_DIVIDER + "\n"
+                + "\tBye. Hope to see you again soon!\n"
+                + "\t" + SECTION_DIVIDER;
+        System.out.println(sign_off);
+    }
+    private static final String COMMAND_WORDS = "list" + "done" + "todo" + "deadline" + "event" + "delete";
+
+    static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void addTask(String taskDescription) throws
+            MissingDeadlineException, MissingEventTimeException {
+        Task t;
+        if (taskDescription.contains("todo")) {
+            if (taskDescription.length() < 5) {
+                throw new IndexOutOfBoundsException();
+            }
+            t = new Todo(taskDescription.substring(5));
+            System.out.println("\tGot it. I've added this task: ");
+        } else if (taskDescription.contains("deadline")) {
+            // Adds task as a Deadline
+            taskDescription.replace("deadline ", "");
+            int byIndex = taskDescription.indexOf("/by ");
+            if (byIndex == -1 || taskDescription.length() < byIndex + 4) {
+                throw new MissingDeadlineException();
+            } else {
+                t = new Deadline(taskDescription.substring(9, byIndex - 1), taskDescription.substring(byIndex + 4));
+            }
+        } else {
+            // Adds task as an Event.
+            taskDescription.replace("event ", "");
+            int atIndex = taskDescription.indexOf("/at ");
+            if (atIndex == -1 || taskDescription.length() < atIndex + 4) {
+                throw new MissingEventTimeException();
+            } else {
+                t = new Event(taskDescription.substring(6, atIndex - 1), taskDescription.substring(atIndex + 4));
+            }
+        }
+        tasks.add(t);
+        System.out.println("\t" + t);
+        System.out.println("\tNow you have " + tasks.size() + " tasks in the list.");
+    }
+
+    public static void markTaskAsDone(String taskDescription){
+        // Marks one of the tasks as done.
+        String[] input = taskDescription.split(" ");        // input = ["Done", taskIndex]
+        int taskIndex = Integer.parseInt(input[1]);
+        try {
+            tasks.get(taskIndex - 1).markAsDone();
+            System.out.println("\t Nice! I've marked this task as done: ");
+            String output = String.format("%02d. [%s] %s", taskIndex, tasks.get(taskIndex - 1).getStatusIcon(), tasks.get(taskIndex - 1).getDescription());
+            System.out.println("\t" + output);
+        } catch (IndexOutOfBoundsException e) {
+            System.out.println("\t Invalid task number!");
+        } catch (NumberFormatException e) {
+            System.out.println("\t Please enter the task number instead!");
+        }
+    }
+
+    public static void listTasks(){
+        // Lists out the things under tasks.
+        System.out.println("\t Here are the tasks in your list:");
+        int i = 0;
+        for(Task t : tasks){
```

Comment:
There is a [for loop standard](https://se-education.org/guides/conventions/java/basic.html#loops) that you could apply, although incrementing the `i` variable seems to do the job.